pages/api/news.ts

```diff
@@ -0,0 +1,26 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next';
+import makeRoute from '../../lib/utils/utils';
+import prisma from '../../prisma/prisma';
+import { NewsWrite, newsWriteSchema } from '../../lib/api/createNewsRequest.dto';
+import { BadRequestError } from '../../lib/utils/errors/badRequestError';
+
+export const getNewsHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  res.status(200).json({ news: await prisma.news.findMany() });
+};
+
+export const postNewsHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const validatedNews = validateCreateNewsRequest(req.body);
+  await prisma.news.create({ data: validatedNews });
+  res.status(201).send('Success!');
```

Comment:
What is the consistent POST behavior across the app- I think this is the first POST endpoint?

Reply:
To clarify, I don't think that we would want to respond with `Success!` for every 201, its not wrong but it means that we would always need to say `Success!`, not really something I would like to enforce everywhere.

Reply:
Gotcha, but is not sending any response problematic? Referring to when Surbhi and I were testing the POST on postman and thought it wasn't succeeding because it timed out due to not sending any response (we realized it worked by going to the endpoint in the browser and seeing what was possted)?

Reply:
I thought `.status` was enough, you might also need `.end` though. i.e.
`res.status(201).end();`

Reply:
@rymaju yup we did it!